### DISCUSSION TITLE
Help~~~~ How to use our model in WebUI to process datasets?

### DISCUSSION THREAD

**Discussion by bl111power** [oobabooga/text-generation-webui#2426] [Q&A]
Hi there,

The models have been installed.
But a Chat dialog can only process one piece of data at a time.
What should I do if I need to process a large amount of data, say more than 100,000 pieces of data or more?
Need help.

Thanks!

**Comment by ossirytk** [comment]
Probably better not to use the webui for that. Sounds like a job for a python script and the terminal. Processing data one by one also seems pretty inefficient. You might want to read the data first, process it into chunks and then call the ai.
